dev-db/postgresql/postgresql16-16.10.recipe

```diff
@@ -127,25 +130,26 @@ PROVIDES_devel="
 	devel:libpq$secondaryArchSuffix = $portVersion
 	"
 REQUIRES_devel="
-	postgresql12$secondaryArchSuffix == $portVersion base
+	postgresql16$secondaryArchSuffix == $portVersion base
 	"
 
 CONFLICTS_server="
 	postgresql
 	postgresql$secondaryArchSuffix
 	postgresql11$secondaryArchSuffix
+	postgresql12$secondaryArchSuffix
```

Comment:
I think this should be `postgresql${secondaryArchSuffix}_server` respectively (for each version).